packages/components/pie-button/README.md

```diff
@@ -60,8 +61,17 @@ import { PieButton } from '@justeattakeaway/pie-button';
 // React
 // For React, you will need to import our React specific component build
 // Which wraps the web component using the @lit-labs/react package
+
+// Note: When using the React version of the component, please make sure
+// you also include React as a dependency in your project as well. See Peer Dependencies section.
+
 import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 ```
+## Peer Dependencies
```

Comment:
I think we may want to add React as an optional dependency. In my opinion I'm not convinced we should have no reference to it in the `package.json` file

Reply:
I spoke to Ash about this. I think leaving it to the consumer makes sense. If they use this component in a vue app then listing it as a peerDep might lead to confusion.